scripts/common.sh

```diff
@@ -84,6 +84,8 @@ function build::common::upload_artifacts() {
 }
 
 function build::gather_licenses() {
+  # force 1.16 since thats the version used to install go-licenses in builder-base
+  build::common::use_go_version 1.18
```

Comment:
Seems to be mismatch here with comment in code. Should it be either 1.16 or 1.18?